### ISSUE TITLE
Trolley's listed as Last Seen: a month ago

### ISSUE THREAD

**Issue by jeremywight** [dacohenii/trolley-tracker-web#5]
@Linktheoriginal @dacohenii Both of the trolleys are listed as last seen: a month ago, and I haven't seen either move today though they are both moving in the iOS app.

**Comment by Linktheoriginal** [comment]
The webapp hadn't been updated to the new API url.  I've updated it and pushed out the change.  I didn't get around to doing the stop time prediction yet, as the Trolley ID returned in the /Trolleys/Running doesn't match the last seen Trolley ID listed on the stop.  (I can work around that with another set of api hits if we want when I get more time.)

I reviewed and tweaked some of the popup styling, but I felt the transparent background made it hard to read.  I do want to take another swing at the schedule popup layout at some point, to sort it by day instead of by route.

At this point, I do most of the work + deploys from the other fork - I haven't merged these in a while.
